subgraphs/saucerswap/v2/src/utils/tick.ts

```diff
@@ -40,19 +40,23 @@ export function createTick(tickId: string, tickIdx: i32, poolId: string, event:
   return tick
 }
 
-export function feeTierToTickSpacing(feeTier: BigInt): BigInt {
+export function feeTierToTickSpacing(feeTier: BigInt): BigInt {  
   if (feeTier.equals(BigInt.fromI32(10000))) {
     return BigInt.fromI32(200)
   }
   if (feeTier.equals(BigInt.fromI32(3000))) {
     return BigInt.fromI32(60)
   }
+  if (feeTier.equals(BigInt.fromI32(1500))) {
```

Comment:
Q: why the need for the extra tier?

Reply:
That is a change that saucerswap confirmed they introduced.